setup/setup.iss

```diff
@@ -31,7 +31,6 @@ WizardStyle=modern
 Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"
 Name: "rfproto"; Description: "Register rf:// protocol handler"; GroupDescription: "Other options:"
 Name: "rflassoc"; Description: "Associate .rfl file extension with Dash Faction Level Editor"; GroupDescription: "Other options:"
-Name: "fftracker"; Description: "Set the multiplayer tracker to rfgt.factionfiles.com"; GroupDescription: "Other options:"

```

Comment:
This option makes sense if Dash Faction is installed after someone saved settings in Red Faction launcher. AFAIK tracker is not changed in DF launcher if the registry key already exist

Reply:
I've noticed that. I assume that behavior is not intentional? Users who install via the .zip distribution may have to manually reset the tracker to the FF one.

Reply:
Why should it remain there if it could be automatically changed by the launcher itself? Most users do not configure this option manually. Imo the installer should only be responsible for the functionality that cannot be handled by the launcher.

Reply:
> Why should it remain there if it could be automatically changed by the launcher itself?

If it were automatically set when the user starts the launcher, regardless of whether the registry key doesn't yet exist or is already set, then sure. But if the installer option were to be just removed as-is with no other changes, based on what you and rafalh have said above, the key will only be set by the launcher if it doesn't already exist. Given the majority of new people install Dash _after_ already having launched the stock game, that key is already going to exist in most cases, so the tracker won't be set to the right value automatically by the launcher (but would have been by the installer).

I see no issues with maintaining the current implementation, however if it were to be removed from the installer, it would be critically important that the launcher first be updated in such a way that ensures a person opening the launcher will be prompted to press a button to set the tracker to rfgt.factionfiles.com before launching the game (whether the key already exists or not). If that isn't done, and the option were just simply removed from the installer, it would lead to player confusion and almost certainly hurt the frequency of would-be new players becoming active in the MP.

I'm not against removing the option necessarily, but it definitely should not be removed from the installer until/unless that functionality had already been migrated to the launcher.

Reply:
Seeing as this issue is becoming a more broad discussion, I have restored the tracker option to setup to separate it from the other changes of this PR. That way we can discuss any further changes to the tracker replacement behavior in #220.